### PR TITLE
Fix: keep data order as it comes from the server

### PR DESCRIPTION
**Pull request by billoosijok**
👉by eliminating normalize()

We thought it was a server bug, but apparently the normalize function messes up the order of the data coming from the server

### CODE REVIEW

**Comment by kdv24** [comment]
As I see it, with this PR:
- data is sorted with the most recently updated appearing at the top of list
- new expenditures or contributions are added to the top
- sorting by date uses the created date to sort both ascending/descending

**Comment by kdv24** [comment]
Looks great, @billoosijok !

File: app/src/state/ducks/contributions.js [hunk 1]
```diff
@@ -222,9 +222,10 @@ export function getContributions(contributionSearchAttrs) {
       const response = await api.getContributions(contributionSearchAttrs);
       if (response.status === 200) {
         const contributions = await response.json();
-        const data = normalize(contributions.data, [schema.contribution]);
```
Comment (danmelton) [inline app/src/state/ducks/contributions.js:225]: @iankerriganharris will this impact our state negatively?

**Comment by kdv24** [comment]
@danmelton don't pull this in yet.  We've got some weirdness on initial loads from the `view` buttons.
On initial click, the url is correct, but the data doesn't load.
Screen Shot 2019-09-11 at 3 24 15 PM

Then on refresh it's all there: 
Screen Shot 2019-09-11 at 3 24 29 PM

File: app/src/state/ducks/contributions.js (hunk 1 above)
Comment (billoosijok) [reply app/src/state/ducks/contributions.js:225]: That's what I'm not sure about. But also if this doesn't get fixed before the presentation it's probably fine.

**Pull request closed without merging**